src/Views/Main.js

```diff
@@ -0,0 +1,86 @@
+import { useState, useEffect } from 'react';
+import { fetchPokemon, fetchTypes } from '../services/FetchPokemon';
+import { TypeSearch } from '../components/Search/TypeSearch';
+import { fetchFilteredPokemon } from '../services/FetchFilteredPokemon';
+import SearchBar from '../components/Search/SearchBar';
+import Sorting from '../components/Sort/Sort';
+import './main.css';
+
+export default function Main() {
+  const [pokemon, setPokemon] = useState([]);
+  const [types, setTypes] = useState([]);
+  const [selectedType, setSelectedType] = useState('all');
+  const [search, setSearch] = useState('');
+  const [sort, setSort] = useState('asc');
+  const [loading, setLoading] = useState(true);
```

Comment:
I see your loading state but your loader isn't displaying. It could be a styling issue, I see you don't have anything within your <span> tags || setTimeout issue.